trunk/SUAVE/Input_Output/VTK/save_vehicle_vtk.py

```diff
@@ -0,0 +1,152 @@
+
+
+def save_vehicle_vtks(vehicle, settings, Results, time_step, prop_filename="propeller.vtk", rot_filename="rotor.vtk",
+                     wake_filename="prop_wake.vtk", wing_filename="wing_vlm.vtk", fuselage_filename="fuselage.vtk", save_loc=None):
+    """
+    Saves SUAVE vehicle components as VTK files in legacy format.
+
+    Inputs:
+       vehicle                Data structure of SUAVE vehicle                    [Unitless] 
+       settings               Settings for aerodynamic analysis                  [Unitless] 
+       Results                Data structure of wing and propeller results       [Unitless]
+       time_step              Simulation time step                               [Unitless]
+       prop_filename          Name of vtk file to save                           [String] 
+       rot_filename           Name of vtk file to save                           [String] 
+       wake_filename          Name of vtk file to save                           [String] 
+       wing_filename          Name of vtk file to save                           [String] 
+       fuselage_filename      Name of vtk file to save                           [String] 
+       save_loc               Location at which to save vtk files                [String]
+       
+    Outputs:                                   
+       N/A
+
+    Properties Used:
+       N/A 
+    
+    Assumptions:
+       Quad cell structures for mesh
+    
+    Source:  
+       None
+    
+    """  
+    # unpack vortex distribution 
+    try:
+        VD = vehicle.vortex_distribution 
+    except:
+        settings = Data()
+        settings.number_spanwise_vortices  = 25
+        settings.number_chordwise_vortices = 5
+        settings.spanwise_cosine_spacing   = False 
+        settings.model_fuselage            = False
+        VD = generate_wing_vortex_distribution(vehicle,settings) 
+        vehicle.vortex_distribution = VD
+    
+    
+    #---------------------------
+    # Save propellers and rotors to vtk
+    #---------------------------
+    for propulsor in vehicle.propulsors:
+        try:
+            print("Attempting to save propeller.")
+            propeller = propulsor.propeller
+            n_props   = int(propulsor.number_of_engines)
+        except:
+            print("No propeller.")
+            n_props = 0
+            
+        if n_props>0:
+            for i in range(n_props):
+                # save the ith propeller
+                if save_loc ==None:
+                    filename = prop_filename
+                else:
+                    filename = save_loc + prop_filename
+                sep  = filename.find('.')
+                file = filename[0:sep]+str(i)+filename[sep:]
+            
+                save_prop_vtk(propeller, file, Results,i, time_step) 
+                
+        try:
+            print("Attempting to save rotor.")
+            rotor = propulsor.rotor
+            n_rots = int(propulsor.number_of_engines)    
```

Comment:
will this break with lift + cruise?

Reply:
No it won't; that is why I split it into two try/except sections so it will try to save the propeller, then try the rotor. For the lift+cruise it generates both the propellers and rotors.

Reply:
I agree with the split, I meant the number of engines:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py#L131

Reply:
ah yes you are right. I updated and tested it with the stopped rotor vehicle so it works now